concepts/structs/about.md

```diff
@@ -1 +1,104 @@
 # Structs
```

Comment:
I'd like to avoid just copy/pasting directly from The Cairo Book. It's a mistake that some of the other concepts are written like that, but that will be changed.

Instead, we should provide a reasonably short, yet clear introduction into what Structs are and how they can be used. If the student wishes to read more, they can then visit TCB themselves.

I really like how [Rust's "Structs" concept](https://github.com/exercism/rust/blob/main/concepts/structs/about.md) is written, can you rewrite this to be similar to that? It's short and sweet. I wouldn't mind if you copy/paste from their concept.
Following Rust's concept format, try to include short examples and descriptions for:
- field init shorthand
- creating instances from other instances with update syntax

Reply:
noted, will make a new PR in a bit